lib/core/app/app_webview.dart

```diff
@@ -87,79 +118,77 @@ class AppBrowser extends InAppBrowser {
 
   @override
   Future onBrowserCreated() async {
-    print("Browser created");
+    appLogger.i("Browser created");
   }
 
   @override
   Future onLoadStart(url) async {
-    print("Started displaying $url");
+    appLogger.i("Started displaying $url");
   }
 
   @override
   Future onLoadStop(url) async {
-    print("Stopped displaying $url");
+    appLogger.i("Stopped displaying $url");
   }
 
   @override
-  void onReceivedError(WebResourceRequest request, WebResourceError error) {
-    print("Can't load ${request.url}.. Error: ${error.description}");
-  }
+  void onReceivedError(WebResourceRequest request, WebResourceError error) =>
+      appLogger.e("Can't load ${request.url}.. Error: ${error.description}",
+          error: error);
 
   @override
   Future<NavigationActionPolicy> shouldOverrideUrlLoading(
       navigationAction) async {
     final url = navigationAction.request.url!;
     if (url.scheme.startsWith("alipay")) {
-      launchUrl(url, mode: LaunchMode.platformDefault,);
+      launchUrl(
+        url,
+        mode: LaunchMode.platformDefault,
+      );
       return NavigationActionPolicy.CANCEL;
     }
     return NavigationActionPolicy.ALLOW;
   }
 
   @override
   void onProgressChanged(progress) {
-    print("Progress: $progress");
+    appLogger.i("Progress: $progress");
   }
 
   @override
   Future<void> onExit() async {
-    print("Browser closed");
+    appLogger.i("Browser closed");
     onClose?.call();
   }
 
-  Future<void> openMacWebview(String url) async {
-    await openUrlRequest(
-            urlRequest: URLRequest(url: WebUri(url)), settings: settings)
-        .then(
-      (value) {
-        print("open mac webview");
-      },
-    );
-  }
+  // navigateWebview navigates to the webview route and displays the given url
+  static Future<void> navigateWebview(BuildContext context, String url) async =>
+      await context.pushRoute(
+        AppWebview(
+          url: url,
+          title: 'lantern_pro_checkout'.i18n,
+        ),
+      );
 
-  static Future<void> openWindowsWebview(String url) async {
-    FlutterWindowsWebview().launchWebview(url);
-  }
+  // openWithSystemBrowser opens a URL in the browser
+  static Future<void> openWithSystemBrowser(String url) async =>
+      await InAppBrowser.openWithSystemBrowser(url: WebUri(url));
 
-  static Future<void> openWebview(String url) async {
+  static Future<void> openWebview(BuildContext context, String url) async {
```

Comment:
There was an issue with opening web view in Linux, I fixed in #1192 , You might need to take pull

Reply:
Thanks! I pulled in those changes